private-previews/identity-bindings/identity-bindings-how-to-guide.md

```diff
@@ -0,0 +1,209 @@
+  az identity create -g $RESOURCE_GROUP -n $MI_NAME
+  ```
+
+2. Create identity binding to map a managed identity to an AKS cluster. 
+
+  ```bash
+  export MI_RESOURCE_ID=$(az identity show -g $RESOURCE_GROUP -n $MI_NAME --query id -o tsv)
+  az aks identity-binding create -g $RESOURCE_GROUP --cluster-name $CLUSTER -n "$MI_NAME-ib" --managed-identity-resource-id $MI_RESOURCE_ID
+  ```
+
+3. AKS creates a unique OIDC issuer per managed identity and populates that in the identity binding resource. Check the identity binding resource to obtain the OIDC issuer URL for this managed identity:
+
+  ```bash
+  az aks identity-binding show -g $RESOURCE_GROUP --cluster-name $CLUSTER -n "$MI_NAME-ib"
+  ```
+
+  Output should be similar to:
+  
+  ```
+  {
+    // omitted other fields
+    "oidcIssuer": {
+          "oidcIssuerUrl": "https://ib.oic.prod-aks.azure.com/<MI-tenant-id>/<MI-client-id>"
+        },
+    // omitted other fields      
+  }
+  ```
+  
+  You should be able to find the FIC with name "aks-identity-binding" created under the managed identity, siimlar to this:
+
+  ![Federated identity credentials created by identity bindings](media/identity-bindings-fic.png)
+
+  Once you see output similar to the above, it confirms that the control plane resources have been successfully created.
+
+## Setup - In-cluster Resource
+
+In this section you will create a cluster level role and role binding to grant your application (service account) to make use of the identity binding for acquiring access token from Entra. You will need to have cluster admin or similar permission to create cluster role / cluster role binding resources.
+
+1. Create ClusterRole and ClusterRoleBinding to authorize the specific namespaces and service accounts that are allowed to use the managed identities (for which identity binding was created earlier):
+
+  ```bash
+  # Obtain credentials needed to access the cluster
+  az aks get-credentials -g $RESOURCE_GROUP -n $CLUSTER -a -f "${CLUSTER}.kubeconfig"
+  export KUBECONFIG="$(pwd)/${CLUSTER}.kubeconfig"
+  export MI_CLIENT_ID=$(az identity show -g $RESOURCE_GROUP -n $MI_NAME --query clientId -o tsv)
+  
+  kubectl apply -f - <<EOF
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: test
+  rules:
+    - verbs: ["use-managed-identity"]
+      apiGroups: ["cid.wi.aks.azure.com"]
+      resources: ["$MI_CLIENT_ID"]
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: test
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: test
+  subjects:
+    - kind: ServiceAccount
+      name: default
+      namespace: default
+  EOF
+  ```
+
+2. Identity binding private preview provides a webhook specifically for the private preview (for public preview, the managed workload identity webhook will add support for identity bindings). Install workload identity webhook using the following step:
```

Comment:
we can probably skip this webhook setup for private preview onboarded clusters. They will be enabled with the private preview version of WI automatically. We can ask the customer to check for the WI version after enabling this.